test/jackknife.jl

```diff
@@ -0,0 +1,88 @@
+@testset "jackknife" begin
+    # Create Jackknife replicate designs
+    bsrs_jk = srs |> jackknifeweights # SRS
+    dstrat_jk = dstrat |> jackknifeweights # Stratified
+    dclus2_jk = dclus2 |> jackknifeweights # Two stage cluster
+    dnhanes_jk = dnhanes |> jackknifeweights # multistage stratified
+    
+    mean_srs_jk = mean([:api00,:api99], bsrs_jk)
+    @test mean_srs_jk.SE[1] ≈ 9.4028 atol = 1e-3
+    @test mean_srs_jk.SE[2] ≈ 9.6575 atol = 1e-3
+    
+    mean_strat_jk = mean([:api00,:api99],dstrat_jk)
+    @test mean_strat_jk.SE[1] ≈ 9.5361 atol = 1e-3
+    @test mean_strat_jk.SE[2] ≈ 10.097 atol = 1e-3
+    
+    mean_clus2_jk = mean([:api00,:api99],dclus2_jk)
+    @test mean_clus2_jk.SE[1] ≈ 34.9388 atol = 1e-3 # R gives 34.928
```

Comment:
In future, check this bug for cluster samples.

Why do R and Julia differ from 2nd decimal place.